docs/zoneddatetime.md

```diff
@@ -1072,7 +1072,30 @@ zdt1.equals(zdt2); // => false (same offset but different time zones)
 zdt1.equals(zdt1); // => true
 ```
 
-### zonedDateTime.**toString**() : string
+### zonedDateTime.**toString**(_options_?: object) : string
+
+**Parameters:**
+
+- `options` (optional object): An object with properties influencing the formatting.
+  The following options are recognized:
+  - `offset` (string): Whether to show the time zone offset in the return value.
+    Valid values are `'auto'` and `'never'`.
+    The default is `'auto'`.
+  - `timeZone` (string): Whether to show the time zone name annotation in the return value.
+    Valid values are `'auto'` and `'never'`.
+    The default is `'auto'`.
+  - `calendar` (string): Whether to show the calendar annotation in the return value.
+    Valid values are `'auto'`, `'always'`, and `'never'`.
+    The default is `'auto'`.
+  - `fractionalSecondDigits` (number or string): How many digits to print after the decimal point in the output string.
+    Valid values are `'auto'`, 0, 1, 2, 3, 4, 5, 6, 7, 8, or 9.
+    The default is `'auto'`.
```

Comment:
Is the behavior of `auto` defined anywhere in the docs? I didn't see it here but might have missed it.

Reply:
Fixed this, I had neglected to copy a few paragraphs from DateTime.